packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -102,14 +118,40 @@ abstract class RenderToggleable extends RenderConstrainedBox implements Semantic
     }
   }
 
+  /// Whether value of this control can be changed by user interaction.
+  ///
+  /// The control is considered interactive if the [onChanged] callback is
+  /// non-null. For example, a non-interactive checkbox is dislayed using a grey
+  /// color and its value cannot be changed.
   bool get isInteractive => onChanged != null;
 
+  /// The visual position of the control.
```

Comment:
position is too confusing a term. It sounds like an x,y coordinate. It's fine for the property name but the doc should say something else. Not sure what exactly, sorry...